### DISCUSSION TITLE
Betreibernamen manuell anpassen

### DISCUSSION THREAD

**Discussion by FunkoLive** [Traewelling/traewelling#2814] [Ideas / Feature Requests]
Hallo zusammen,

in diesem Bild

![Screenshot 2024-08-06 at 15-13-16 Statistiken - Träwelling](https://github.com/user-attachments/assets/2a86c5d5-feea-45ad-8315-b2223cd41b53)

sieht man, dass in den Statistiken ein Betreibername doppelt aufgelistet wurde, da er auf manchen Linien nur "Rhein-Neckar-Verkehr GmbH" heißt und auf manchen "Rhein-Neckar-Verkehr GmbH (Oberrheinische Eisenbahn)". Hier wäre es sinnvoll, wenn man den Betreibernamen manuell anpassen oder Gruppen bilden könnte, zu denen man die jeweiligen Betreibernamen zusammenfassen kann.

Grüße
Felix

**Comment by kpleiss** [comment]
Hallo,

finde ich eine gute Idee. Es hätte zusätzlich den Vorteil, dass man bei nicht hinterlegten Betreibern diese ergänzen könnte, ohne den Statustext zu benutzen.

VG

**Comment by MrKrisKrisu** [comment]
Das geht mittlerweile prizipiell über den Umweg dieses Repos:
https://github.com/Traewelling/transitous-wikidata-operator-matching

Die operators.csv wird regelmäßig eingelesen und die Namen in der Datenbank damit aktualisiert.
Voraussetzung allerdings: Der Betreiber muss dort gemapped sein. (mapping.csv)
